docs/subnets/customize-a-subnet.md

```diff
@@ -0,0 +1,708 @@
+* [subnet-cli](./subnet-cli.md#subnet-cli-create-blockchain)
+
+
+### Setting the Genesis Allocation
+
+
+Alloc defines addresses and their initial balances. This should be changed accordingly for each chain. If you don't
+provide any genesis allocation, you won't be able to interact with your new
+chain (all transactions require a fee to be paid from the sender's balance). 
+
+The `alloc` field expects key-value pairs. Keys of each entry must be a valid `address`. The `balance` field in the value can be either a `hexadecimal` or `number` to indicate initial balance of the address. The default value contains `8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC` with `50000000000000000000000000` balance in it. Default:
+
+```json
+  "alloc": {
+    "8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC": {
+      "balance": "0x295BE96E64066972000000"
+    }
+  }
+```
+
+
+To specify a different genesis allocation, populate the `alloc` field in the genesis JSON as follows:
+
+```json
+  "alloc": {
+    "8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC": {
+      "balance": "0x52B7D2DCC80CD2E4000000"
+    },
+    "Ab5801a7D398351b8bE11C439e05C5B3259aeC9B": {
+      "balance": "0xa796504b1cb5a7c0000"
+    }
+  },
+```
+
+The keys in the allocation are [hex](https://en.wikipedia.org/wiki/Hexadecimal) addresses **without the canonical `0x` prefix**.
+The balances are denominated in Wei ([10^18 Wei = 1 Whole Unit of Native Token](https://eth-converter.com/)) and expressed as
+hex strings **with the canonical `0x` prefix**. You can use [this converter](https://www.rapidtables.com/convert/number/hex-to-decimal.html)
+to translate between decimal and hex numbers.
+
+The above example yields the following genesis allocations (denominated in whole units of the native token ie. 1 AVAX/1 WAGMI):
+
+```
+0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC: 100000000 (0x52B7D2DCC80CD2E4000000=100000000000000000000000000 Wei)
+0xAb5801a7D398351b8bE11C439e05C5B3259aeC9B: 49463 (0xa796504b1cb5a7c0000=49463000000000000000000 Wei)
+```
+
+### Setting a Custom Fee Recipient
+
+By default, all fees are burned (sent to the blackhole address with `"allowFeeRecipients": false`). However, it is
+possible to enable block producers to set a fee recipient (who will get compensated for blocks they produce).
+
+To enable this feature, you'll need to add the following to your
+genesis file (under the `"config"` key):
+
+```json
+{
+  "config": {
+    "allowFeeRecipients": true
+  }
+}
+```
+
+#### `feeRecipient` 
+
+With `allowFeeRecipients` enabled, your validators can specify their addresses to collect fees. They need to update their EVM   [chain config](#feerecipient-1) with the following to specify where the fee should be sent to.
+
+```json
+{
+  "feeRecipient": "<YOUR 0x-ADDRESS>"
+}
+```
+
+
+:::warning
+
+If `allowFeeRecipients` feature is enabled on the subnet, but a validator doesn't specify a "feeRecipient", the fees will be burned in blocks it produces.
+
+:::
+
+
+### Build Genesis
```

Comment:
We can probably remove this, the buildGenesis API is pretty useless. If you already have the JSON all it does is marshal it and change it to cb58.

Reply:
Removed